pennylane_lightning/core/src/observables/tests/Test_Observables.cpp

```diff
@@ -274,9 +285,15 @@ template <typename TypeList> void testTensorProdObsBase() {
                 VectorT expected =
                     createProductState<PrecisionT, ComplexT>("0+1");
 
+#ifdef _ENABLE_PLGPU
+                REQUIRE(isApproxEqual(state_vector.getDataVector().data(),
+                                      state_vector.getDataVector().size(),
+                                      expected.data(), expected.size()));
+#else
                 REQUIRE(isApproxEqual(state_vector.getData(),
                                       state_vector.getLength(), expected.data(),
                                       expected.size()));
+#endif
```

Comment:
I wonder if there is an API design that can avoid these separations in call signature.

Reply:
Fixed after we add getDataVector() to LQubitRaw